### PR TITLE
Set default value for vsock_test_tool

### PR DESCRIPTION
**Pull request by qcheng-redhat**
Set default value for vsock_test_tool

Signed-off-by: qcheng-redhat <qcheng@redhat.com>

ID: 2500

### CODE REVIEW

**Comment by qcheng-redhat** [comment]
Test results: 

(01/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (01/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (682.01 s)
 (02/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_powerdown.q35: STARTED
 (02/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_powerdown.q35: PASS (103.27 s)
 (03/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_reset_bootable.q35: STARTED
 (03/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_reset_bootable.q35: PASS (374.45 s)
 (04/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_reset_during_boot.q35: STARTED
 (04/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.system_reset_during_boot.q35: PASS (636.19 s)
 (05/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.reboot.q35: STARTED
 (05/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.reboot.q35: PASS (1260.39 s)
 (06/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.boot.q35: STARTED
 (06/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.boot.q35: PASS (61.28 s)
 (07/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.stop_continue.stop_cont_only.q35: STARTED
 (07/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.stop_continue.stop_cont_only.q35: PASS (68.44 s)
 (08/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.stop_continue.with_file_copy.q35: STARTED
 (08/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.stop_continue.with_file_copy.q35: PASS (77.47 s)
 (09/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.shutdown.q35: STARTED
 (09/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.shutdown.q35: PASS (59.69 s)
 (10/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.q35: STARTED
 (10/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.q35: PASS (154.09 s)
 (11/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.negative_test.q35: STARTED
 (11/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.negative_test.q35: PASS (129.62 s)
 (12/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: STARTED
 (12/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (189.58 s)
 (13/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.migration_with_vsock.q35: STARTED
 (13/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.migration_with_vsock.q35: PASS (417.85 s)
 (14/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.iommu_enabled.q35: STARTED
 (14/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.iommu_enabled.q35: PASS (197.22 s)
 (15/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.vsock_test_suite.q35: STARTED
 (15/15) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.vsock_test.vsock_test_suite.q35: PASS (84.88 s)

**Comment by qcheng-redhat** [comment]
@XueqiangWei Can you help review this patch, when you have time. Thanks!

**Comment by qcheng-redhat** [comment]
Hi, @ybduan please help review nested related case change. Thanks.

**Comment by ybduan** [comment]
(08/10) vsock_con_sockets.Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.nested_vsock_con_sockets.q35: STARTED
 (08/10) vsock_con_sockets.Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.10.0.x86_64.io-github-autotest-qemu.nested_vsock_con_sockets.q35: PASS (408.46 s)

LGTM.
Aced-by: Yanbin Duan <yduan@redhat.com>

**Comment by qcheng-redhat** [comment]
@fbq815 Please help review this patch on S390, thanks!

**Comment by qcheng-redhat** [comment]
Hi @BohdanMar, could you help review this patch, please? Thanks!

**Comment by BohdanMar** [comment]
Hello @qcheng-redhat 
nested_vsock_con_sockets testcase doesn't support on aarch64 platform

**Comment by fbq815** [comment]
LGTM, the test could PASS on s390x

**Comment by yanan-fu** [comment]
To make it adopt with the version in the future,   suggest set the new vsock_test_tool as default.

**Comment by qcheng-redhat** [comment]
> To make it adopt with the version in the future, suggest set the new vsock_test_tool as default.

Updated. @yanan-fu

File: qemu/tests/cfg/nested_vsock_con_sockets.cfg [hunk 1]
```diff
@@ -10,8 +10,7 @@
     vsocks = vhost_vsock0
     vsock_port = 2345
     no RHEL.7, RHEL.8.0, RHEL.8.1, RHEL.8.2
-    RHEL.8:
-      vsock_test_tool = nc_vsock
-    RHEL.9:
-      vsock_test_tool = ncat
+    vsock_test_tool = nc_vsock
+    ! RHEL.8:
+        vsock_test_tool = ncat
```
Comment (yanan-fu) [inline qemu/tests/cfg/nested_vsock_con_sockets.cfg:15]: Suggest:
```
vsock_test_tool = ncat
RHEL.8:
    vsock_test_tool = nc_vsock
```

Means the default one is `ncat`, for rhel8 specific is `nc_vsock`.

Comment (qcheng-redhat) [reply qemu/tests/cfg/nested_vsock_con_sockets.cfg]: update.

**Review by yanan-fu — APPROVED** [review]
Ack

**Pull request merged**